src/pages/bootcamp-dimitar/index.tsx

```diff
@@ -0,0 +1,214 @@
+import * as React from 'react'
+import { GetServerSideProps } from 'next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { useBootcampDimitarList } from '../../common/hooks/bootcampDimitar'
+import { DataGrid, GridColumns, GridRowId } from '@mui/x-data-grid'
+import CustomLayout from '../../components/bootcamp-dimitar/CustomLayout'
+import EditIcon from '@mui/icons-material/Edit'
+import Link from 'next/link'
+import { Button, IconButton } from '@mui/material'
+import PageviewIcon from '@mui/icons-material/Pageview'
+import DeleteIcon from '@mui/icons-material/Delete'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import Modal from '@mui/material/Modal'
+import { deleteBootcampDimitar } from '../../common/rest'
+import { useRouter } from 'next/router'
+import { BootcampDimitarResponse } from 'gql/bootcampDimitar'
+import DeleteSelectedModal from 'components/bootcamp-dimitar/DeleteSelectedModal'
+import PersonAddIcon from '@mui/icons-material/PersonAdd'
+import { useMutation } from 'react-query'
+import { ApiErrors } from 'common/api-errors'
+import { AlertStore } from 'stores/AlertStore'
+import { AxiosError, AxiosResponse } from 'axios'
```

Comment:
The imports should be sorted in three separate groups by their location with an empty line as a separator:

- external imports
- absolute imports
- relative imports

```suggestion
import Link from 'next/link'
import * as React from 'react'
import Box from '@mui/material/Box'
import { useRouter } from 'next/router'
import Modal from '@mui/material/Modal'
import { useMutation } from 'react-query'
import { GetServerSideProps } from 'next'
import EditIcon from '@mui/icons-material/Edit'
import { AxiosError, AxiosResponse } from 'axios'
import Typography from '@mui/material/Typography'
import { Button, IconButton } from '@mui/material'
import DeleteIcon from '@mui/icons-material/Delete'
import PageviewIcon from '@mui/icons-material/Pageview'
import PersonAddIcon from '@mui/icons-material/PersonAdd'
import { DataGrid, GridColumns, GridRowId } from '@mui/x-data-grid'
import { serverSideTranslations } from 'next-i18next/serverSideTranslations'

import { ApiErrors } from 'common/api-errors'
import { AlertStore } from 'stores/AlertStore'
import { BootcampDimitarResponse } from 'gql/bootcampDimitar'
import DeleteSelectedModal from 'components/bootcamp-dimitar/DeleteSelectedModal'

import { deleteBootcampDimitar } from '../../common/rest'
import CustomLayout from '../../components/bootcamp-dimitar/CustomLayout'
import { useBootcampDimitarList } from '../../common/hooks/bootcampDimitar'
```

Also some of the imports can be combined such as:

```
import DeleteIcon from '@mui/icons-material/Delete'
import PageviewIcon from '@mui/icons-material/Pageview'
import PersonAddIcon from '@mui/icons-material/PersonAdd'
```

```
import { Delete, Pageview, PersonAdd } from '@mui/icons-material'
```